### PR TITLE
Examiner Dashboard Updates

### PR DESCRIPTION
**Pull request by dimak1**
*Issue:*

*Description of changes:*
- Refactor and update Examiner Dashboard
- Add layout, update UI, remove breadcrumbs, etc.
- Improve tab navigation

Screenshot 2024-12-20 at 14 59 22

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: strr-examiner-web/app/app.config.ts
Comment (deetz99) [inline strr-examiner-web/app/app.config.ts:1]: The allowed idps in the header should also be set to idir btw. Full connect config here, subject to change.

```
connect: {
    core: {
      login: {
        redirectPath: '',
        idps: () => []
      },
      header: {
        options: {
          localeSelect: false,
          unauthenticated: {
            whatsNew: false,
            loginMenu: false,
            createAccount: false
          },
          authenticated: {
            notifications: true,
            accountOptionsMenu: true
          }
        }
      },
      plugin: {
        authApi: {
          errorRedirect: {
            401: '/auth/login'
          }
        },
        payApi: {
          errorRedirect: {
            401: '/auth/login'
          }
        }
      }
    }
  },
```

File: strr-examiner-web/app/layouts/examiner.vue [hunk 1]
```diff
@@ -1,37 +1,50 @@
...
+  }
+
   return navigateTo(localePath(navigateToPath))
 }
 
 </script>
 <template>
   <div>
-    <ConnectHeaderWrapper>
+    <ConnectHeaderWrapper
```
Comment (deetz99) [inline strr-examiner-web/app/layouts/examiner.vue:45]: If you need a change to this at all I can get a new version out relatively quickly.

File: strr-examiner-web/app/app.config.ts
Comment (dimak1) [reply strr-examiner-web/app/app.config.ts:1]: Ok, let me update it. Thanks!

**Review by deetz99 — APPROVED** [review]
looks good to me 👍

File: strr-examiner-web/app/layouts/examiner.vue (hunk 1 above)
Comment (dimak1) [reply strr-examiner-web/app/layouts/examiner.vue:45]: You mean if I need to adjust anything in this component? Yea, sounds good. So far ok, just some style updates, but if anything, I let you know, thanks!

Comment (deetz99) [reply strr-examiner-web/app/layouts/examiner.vue:45]: If the padding or whatever is causing an issue.

**Pull request merged**